components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -188,19 +188,26 @@ export class Panel extends React.Component<Props, State> {
     }
   }
 
-  onBackupWallet = (id: string) => {
-    chrome.tabs.create({
-      url: 'chrome://rewards#manage-wallet'
-    })
-    this.actions.deleteNotification(id)
+  onEvent = (learnMore: string) => {
+    return function (id: string) {
+      chrome.tabs.create({
+        url: learnMore
+      })
+      this.actions.deleteNotification(id)
```

Comment:
Because this `this` is contained in a `function` expression, it won't necessarily be the `this` of the surrounding lexical context (although I think it happens to be working out like you want at the moment). Instead of using a `function` expression, we should use an arrow function:

```ts
return (id:string) => {
  ....
}
```